gooddata-pandas/gooddata_pandas/result_convertor.py

```diff
@@ -11,14 +14,179 @@
 LabelOverrides = dict[str, dict[str, dict[str, str]]]
 
 
+class _Header(ABC):
```

Comment:
i think this should also have `__slots__ = ()` so as not to pollute class hierarchy.

see: https://stackoverflow.com/questions/1816483/how-does-inheritance-of-slots-in-subclasses-actually-work